src/main/resources/Quests.json

```diff
@@ -1,20 +1,17 @@
-{
-  "Quest": {
```

Comment:
Prefer these headings left in, as when the file is full each object will be the name of the quest/mob/shop etc

Reply:
You can't have headings for objects in arrays.

Reply:
Okay, was originally just gonna store all quest objects within an object, but guess an array doesn't hurt it much. 
There any large advantage of using an array that we gain from losing object names of each object?

Reply:
It'd simplify the data structure. When the JSON parsing library parses the file, it tries to store the different fields of the object in variables with the same name, eg if there is a field called questName, it'll store the fields value in the variable questName if the output class has a variable named that. If we keep the headers, the class that we'd parse to would have to have a different variable for each quest, shop, chunk ect.

Reply:
Ok fair enough